src/project/internal/exportprojectscenario.cpp

```diff
@@ -176,7 +176,6 @@ bool ExportProjectScenario::exportScores(const notation::INotationPtrList& notat
 
     INotationWriter::Options options {
         { INotationWriter::OptionKey::UNIT_TYPE, Val(unitType) },
-        { INotationWriter::OptionKey::TRANSPARENT_BACKGROUND, Val(imagesExportConfiguration()->exportPngWithTransparentBackground()) }
```

Comment:
I realised one more thing: if PngWriter (and SvgWriter) don't actually use the `TRANSPARENT_BACKGROUND` OptionKey anymore, it is better to remove it entirely, from `INotationWriter` and `IProjectWriter`, to avoid confusion.

_But_ it is still used in `BackendApi::exportScorePngs`, which is for use in the MuseScore.com backend. For the backend, it is quite important that the 'transparent background' option is off, regardless of whatever value may be in the configuration. 

The best solution might be the following:
- Remove the option here from ExportProjectScenario, but let it remain in other places (i.e. exactly like what you have done now)
- In PngWriter/SvgWriter, check if this option is set; if yes, use the value of the option; else, use the value from Configuration.